server/lib/utils/logs.ts

```diff
@@ -72,17 +74,27 @@ export function logError (err: ContextualizedError, label?: string) {
     return
   }
 
+  reduceForwardedErrorsVerbosity(err)
   log(err, label, 'red')
 
-  const url = err?.context?.url
-  const host = isUrl(url) ? getHost(url) : 'local'
+  let host = 'local'
+  if ('context' in err) {
+    const { host: errHost, url } = err.context
+    host = (errHost as Host) || (isAbsoluteUrl(url) ? getHost(url as AbsoluteUrl) : 'local')
+  }
   const errorStatusCode = err.statusCode || 500
   countsByHostAndErrorStatusCode[host] ??= {}
   countsByHostAndErrorStatusCode[host][errorStatusCode] ??= 0
   countsByHostAndErrorStatusCode[host][errorStatusCode]++
   errorCount++
 }
 
+function reduceForwardedErrorsVerbosity (err: ContextualizedError) {
```

Comment:
```suggestion
function decreaseForwardedErrorsVerbosity (err: ContextualizedError) {
```
as `reduce` is ambiguous

Reply:
done in 978067ce0